ui-tests/src/test/java/org/openmrs/reference/page/MarkPatientDeceasedPage.java

```diff
@@ -0,0 +1,124 @@
+    private static final By ADD_ANSWER_BUTTON = By.cssSelector("#codedDatatypeRow > td > table > tbody > tr > td.buttons > input:nth-child(1)");
+    private static final By CAUSE_SEARCH_FIELD = By.id("newAnswerConcept");
+    private static final By FIRST_CAUSE_OPTION = By.className("autocompleteresult");
+    private static final By ADD_BUTTON = By.className("ui-button-text");
+    private static final By SAVE_CONCEPT_BUTTON = By.cssSelector("#saveDeleteButtons > input[type=submit]:nth-child(1)");
+    private static final By SEARCH_CONCEPT_FIELD = By.id("inputNode");
+    private static final By FIRST_RESULT = By.className("odd");
+    private static final By CONCEPT_ID = By.cssSelector("#conceptTable > tbody > tr:nth-child(1) > td");
+    private static final By ADMINISTRATION_PAGE = By.id("administrationNavLink");
+    private static final By ADVANCED_SETTINGS = By.linkText("Advanced Settings");
+    private static final By CONCEPT_VALUE_FIELD = By.cssSelector("#globalPropsList > tr:nth-child(133) > td:nth-child(2) > input[type=text]");
+    private static final By SAVE_CONCEPT_VALUE = By.name("action");
+    private static final By HOME_LINK = By.id("homeNavLink");
+    private static final By MARK_PATIENT_DECEASED_LINK = By.id("org.openmrs.module.coreapps.markPatientDead");
+    private static final By MARK_PATIENT_DECEASED_CHECK_BOX = By.id("deceased");
+    private static final By DATE_PICKER = By.id("death-date-display");
+    private static final By SAVE_DECEASED_BUTTON= By.cssSelector("#mark-patient-dead > fieldset > p:nth-child(4) > span:nth-child(2) > input");
+    private static final By ACTIVE_DAY = By.cssSelector("td.day.active");
+    private static final By DEATH_MESSAGE_TEXT = By.className("death-message");
+    private static final By EDIT_CONCEPT_LINK = By.id("editConcept");
+    private static final By DELETE_BUTTON = By.cssSelector("#saveDeleteButtons > input[type=submit]:nth-child(4)");
+    private static final By DELETE_CONFIRMATION = By.id("openmrs_msg");
+
+    public MarkPatientDeceasedPage(Page page) {
+        super(page);
+    }
+
+    public void getConcept() {
+        clickOn(DICTIONARY_LINK); 
+        Concept();
+        clickOn(HOME_LINK); 
+     
+    }
+
+    /*
+     * If there is a concept called Cause Of Death, The method gets its concept code
+     * if there is no concept at all or no concept called Cause of death, the method
+     * creates one and copies its concept code
+     */
+    
+    public void Concept() {
+        newConcept();
+        enterConceptid(getConceptId());
+    }
+
+    public void clickOnMarkPatientDead() {
+        clickOn(MARK_PATIENT_DECEASED_LINK); 
+        clickOn(MARK_PATIENT_DECEASED_CHECK_BOX);
+        clickOn(DATE_PICKER);
+        pickDate();
+        Select dropdown = new Select(findElement(By.id("cause-of-death")));
+        dropdown.selectByValue("113230AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA");
+        findElement(SAVE_DECEASED_BUTTON).click();
```

Comment:
What about using `clickOn(SAVE_DECEASED_BUTTON)` ?

Reply:
i have fixed this